instaloader/instaloadercontext.py

```diff
@@ -508,12 +509,21 @@ def graphql_query(self, query_hash: str, variables: Dict[str, Any],
 
             variables_json = json.dumps(variables, separators=(',', ':'))
 
-            resp_json = self.get_json('graphql/query',
-                                      params={'query_hash': query_hash,
-                                              'variables': variables_json},
-                                      session=tmpsession)
+            # XXX Dirty fix here
+            if query_hash == "2b0673e0dc4580674a88d426fe00ea90":
+                resp_json = self.get_json('graphql/query',
+                                          params={'doc_id': '8845758582119845',
+                                                  'variables': variables_json},
+                                          session=tmpsession)
+
+            else:
+                resp_json = self.get_json('graphql/query',
+                                          params={'query_hash': query_hash,
+                                                  'variables': variables_json},
+                                          session=tmpsession)
```

Comment:
I noticed that the `graphql_query` function has a nearly identical counterpart, `doc_id_graphql_query`, which takes a `doc_id` instead of a `query_hash`. Since the query hash `2b0673e0dc4580674a88d426fe00ea90` is only referenced in two places within the codebase, we can just swap out those `graphql_query` calls with `doc_id_graphql_query`. What do you think?

```py
@classmethod
def from_mediaid(cls, context: InstaloaderContext, mediaid: int):
    """Create a StoryItem object from a given mediaid.

    .. versionadded:: 4.9
    """
    # Before -----
    pic_json = context.graphql_query(
        '2b0673e0dc4580674a88d426fe00ea90',
        {'shortcode': Post.mediaid_to_shortcode(mediaid)}
    )
    # After -----
    pic_json = context.doc_id_graphql_query(
        '8845758582119845',
        {'shortcode': Post.mediaid_to_shortcode(mediaid)}
    )
```